routes/commons.go

```diff
@@ -0,0 +1,14 @@
+// Package routes defines all API's grouped by each file.
+// TODO: Access Control
+package routes
+
+import "github.com/julienschmidt/httprouter"
+
+// Route struct that stores info about a route
+type Route struct {
+	Name    string
+	Method  string
+	Pattern string
+	Handler httprouter.Handle
```

Comment:
FYI in node (express.js) the handler could be an array of midwares. In that case it is easy to chain `jwt-auth` middle ware in the front.